APIs/AuthorizationAPI.raml

```diff
@@ -190,30 +195,35 @@ types:
       example: "token"
   post:
     description: |
-      Method for requesting OAuth 2.0 Bearer Token (Authorization Code, Client Credentials and Refresh Grants).
+      Method for requesting OAuth 2.0 Bearer Token (Authorization Code, Client Credentials and Refresh Token grants).
 
-      **Note**: The data flow is dependent on the chosen grant type (e.g. authorization_code, refresh grant, etc.)
+      **Note**: The data flow is dependent on the chosen grant type (e.g. authorization_code, refresh_token, etc.)
 
       **Note**: Request examples may be rendered as JSON objects, but should be passed in as "application/x-www-form-urlencoded"
       MIME type, in accordance with [RFC 6749](https://tools.ietf.org/html/rfc6749).
     securedBy: [ basic, private ]
     is: ["OAuthBasic", "OAuthPrivate"]
     body:
       application/x-www-form-urlencoded:
-        type: TokenAuthorizeRequest | TokenRefreshRequest
+        type: TokenAuthorizeRequest | TokenCredentialsRequest | TokenRefreshRequest
         examples:
           AuthCodeGrant:
             grant_type: authorization_code
             code: SplxlOBeZQQYbYS6WxSbIA
             redirect_uri: "http://client.example.com/"
             client_id: ttwCD9hb4OckmcGDnmC3J3Jr
+          ClientCredentialsGrant:
+            grant_type: client_credentials
+            client_id: c1068569-80c1-48cf-982e-30c3f930dfe3
```

Comment:
Would be useful to show the client ID and client secret being sent in the Body of the request in the RAML also, as its permitted in RFC6749

Reply:
may be we give different examples, one for using private_key_jwt, and one without.  The one without will require to send client ID and client secret.

Reply:
As things stand, the method of sending the client_id and client_secret in the request body isn't explicitly called out by this specification (ie. `client_secret_post`). We could do, but I'm not sure this has an obvious benefit over `client_secret_basic` unless your HTTP stack is very limited, and we're preferring use of `private_key_jwt` anyway.